js/main.js

```diff
@@ -175,4 +186,65 @@ function showEntryForm(event) {
       $previewPhoto.setAttribute('src', $imageContent.getAttribute('src'));
     }
   }
+
+  // delete button in edit page
+  var $deleteButtonRow = document.querySelector('.form-actions');
+  var $deleteModalButton = document.createElement('button');
+  $deleteButtonRow.prepend($deleteModalButton);
+  $deleteModalButton.setAttribute('type', 'button');
+  $deleteModalButton.setAttribute('class', 'modal');
+  $deleteModalButton.textContent = 'Delete Entry';
+  $deleteButtonRow.setAttribute('class', 'form-actions-edit column-full');
+
+  // delete button hovering
+  $deleteModalButton.addEventListener('mouseover', mouseoverModal);
+  $deleteModalButton.addEventListener('mouseout', mouseoutModal);
+
+  function mouseoverModal(event) {
+    $deleteModalButton.className = 'modal-hovered';
+  }
+
+  function mouseoutModal(event) {
+    $deleteModalButton.className = 'modal';
+  }
+
+  // show modal
+  $deleteModalButton.addEventListener('click', showModal);
+
+  var deleteModalShowing = false;
+  var $modalBackground = document.querySelector('.modal-background');
+
+  function showModal(event) {
+    if (deleteModalShowing === true) {
+      $modalBackground.className = 'modal-background hidden';
+      deleteModalShowing = false;
+    } else {
+      $modalBackground.className = 'modal-background';
+      deleteModalShowing = true;
+    }
+  }
+
+  // cancel and confirm button functions
+  var $cancelButton = document.querySelector('.cancel-button');
+  var $confirmButton = document.querySelector('.confirm-button');
+
+  $cancelButton.addEventListener('click', showModal);
+  $confirmButton.addEventListener('click', confirmButton);
+
+  function confirmButton(event) {
+    for (var i = 0; i < allEntries.length; i++) {
+      if (data.editing === allEntries[i].getAttribute('data-entry-id')) {
+        var $currentLi = allEntries[i];
+        $currentLi.setAttribute('class', 'hidden');
```

Comment:
You should delete the element entirely, not just hide it. You can use the `remove` method